### ISSUE TITLE
[🐞]: The SafeSack bug (Safe and Sack tooltips only display the name)

### ISSUE THREAD

**Issue by Egassy** [MehVahdJukaar/Supplementaries#1294]
### Before Continuing:

- [X] Check you are using the latest version of the mods and its dependencies
- [X] Always include a latest.log if you are crashing
- [X] Remove mod that enhances Minecraft: Optifine, Sodium, others. The issue persists.
- [X] If you are unsure which mod is the culprit.
Disable all of your mods and enable them 1-2 mods each time to isolate the culprit
- [X] Confirm that there is no existing issue with a similar description submitted in the list of issues.

### Version

1.20.1-FORGE

### Supplementaries Version

supplementaries-1.20-3.1.11

### Moonlight Lib Version

moonlight-1.20-2.13.41-forge

### Describe Issue

With Quark installed, the Safe and the Sack have no tooltip other than their name. Advanced tooltips and mod name tooltip are both removed.

From my limited understanding of the code, it is likely to be the area around line 69 of [QuarkClientCompatImpl](https://github.com/MehVahdJukaar/Supplementaries/blob/master/forge/src/main/java/net/mehvahdjukaar/supplementaries/integration/forge/QuarkClientCompatImpl.java#L69) that is causing the issue as it appears to remove any string that is formatted with `§`, and as far as I'm aware the only fully unformatted text in a tooltip is the item name.

![image](https://github.com/user-attachments/assets/8ee2cc74-5f95-4f58-b28c-2a4b82b55dfa)
![image](https://github.com/user-attachments/assets/c9cd79f3-eafa-4aa2-bf4a-bcc033bfe25b)

Modlist when testing is
```emi-1.1.18+1.20.1+forge
jei-1.20.1-forge-15.20.0.105
moonlight-1.20-2.13.41-forge
Quark-4.0-460
supplementaries-1.20-3.1.11
Zeta-1.0-24```

**Comment by MehVahdJukaar** [comment]
isnt this same as what shulker boxes do?

**Comment by Egassy** [comment]
No, because Shulker Boxes do not remove the vanilla debug tooltip (containing itemid) or the modded mod name tooltip.
